pkg/splunk/enterprise/monitoringconsole.go

```diff
@@ -313,16 +367,21 @@ func AddURLsConfigMap(revised *corev1.ConfigMap, crName string, newURLs []corev1
 }
 
 //DeleteURLsConfigMap for deleting server peers to the monitoring console or scaling down
-func DeleteURLsConfigMap(revised *corev1.ConfigMap, crName string, newURLs []corev1.EnvVar, deleteCR bool) {
+func DeleteURLsConfigMap(revised *corev1.ConfigMap, crName string, newURLs []corev1.EnvVar, deleteCR bool) []string {
+	var deletedPeers []string
 	for _, url := range newURLs {
 		currentURLs := strings.Split(revised.Data[url.Name], ",")
```

Comment:
Nit: Just a thought, are we making sure the **configMap revised's data section** has an entry for **key url.Name**? If yes, please ignore.

Reply:
yes, revised configMap is always updated